lib/load-config.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+const requireEsm = require('esm')(module);
+const path = require('path');
+const fs = require('fs');
+const pkgConf = require('pkg-conf');
+
+const loadConfigFile = projectDir => {
+	const loaded = requireEsm(path.resolve(projectDir, 'ava.config.js'));
+	const config = (loaded.__esModule) ? loaded.default : loaded;
+	if (config.then) {
+		throw new Error('Config file must not return a promise');
+	}
+	return (typeof config === 'function') ? config({projectDir}) : config;
```

Comment:
The `config({projectDir})` return value *also* must not be a promise. Happy to fix this up myself.